### PR TITLE
Update platform naming

### PR DESCRIPTION
**Pull request by WilliamBergamin**
###  Summary

This PR aims to update the project to follow the new Slack Term Glossary, this mainly affect documentation and comments, variable names were left unchanged to avoid breaking changes.

Some liberty was taken to reduce the number of proper nouns used in headers and titles (ex: Events -> events) but this was not applied everywhere, lmk if this PR should include a more extensive update of these terms

### Requirements 

* [x] I've read and understood the [Contributing Guidelines](https://github.com/slackapi/deno-slack-sdk/blob/main/.github/CONTRIBUTING.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).

### CODE REVIEW

**Review by jesselawson — APPROVED** [review]
dab.jpeg

**Review by shapirone — CHANGES_REQUESTED** [review]
Mostly small comments, opened a discussion in Slack regarding using `*Fn` as a naming convention for function signature types (which would help us avoid the potentially dangerous name update)

File: src/functions/interactivity/view_router_test.ts [hunk 1]
```diff
@@ -114,8 +113,7 @@ type SlackViewSubmissionHandlerTesterFn = {
   >;
 };
 
-type SlackViewClosedHandlerTesterFn = {
-  // Accept a Slack Function
+type SlackViewClosedHandlerTesterType = {
```
[the remark this answers is not in the document]
Comment (WilliamBergamin) [reply src/functions/interactivity/view_router_test.ts]: `Type` changed in favor of `Signature`

File: src/functions/interactivity/view_router_test.ts [hunk 2]
```diff
@@ -94,8 +94,7 @@ type SlackViewClosedHandlerTesterResponse<
   >;
 };
 
-type SlackViewSubmissionHandlerTesterFn = {
-  // Accept a Slack Function
+type SlackViewSubmissionHandlerTesterType = {
```
[the remark this answers is not in the document]
Comment (WilliamBergamin) [reply src/functions/interactivity/view_router_test.ts]: `Type` changed in favor of `Signature`

File: src/functions/interactivity/block_suggestion_router_test.ts [hunk 3]
```diff
@@ -51,8 +51,7 @@ type SlackSuggestionHandlerTesterResponse<
   createContext: CreateSuggestionHandlerContext<InputParameters, RequiredInput>;
 };
 
-type SlackSuggestionHandlerTesterFn = {
-  // Accept a Slack Function
+type SlackSuggestionHandlerTesterType = {
```
[the remark this answers is not in the document]
Comment (WilliamBergamin) [reply src/functions/interactivity/block_suggestion_router_test.ts]: `Type` changed in favor of `Signature`

File: src/functions/interactivity/block_actions_router_test.ts [hunk 4]
```diff
@@ -50,8 +50,7 @@ type SlackActionHandlerTesterResponse<
   createContext: CreateActionHandlerContext<InputParameters, RequiredInput>;
 };
 
-type SlackActionHandlerTesterFn = {
-  // Accept a Slack Function
+type SlackActionHandlerTesterType = {
```
[the remark this answers is not in the document]
Comment (WilliamBergamin) [reply src/functions/interactivity/block_actions_router_test.ts]: `Type` changed in favor of `Signature`

**Review by shapirone — APPROVED** [review]
🚢

**Pull request merged**